components/html-block/html-block.js

```diff
@@ -131,14 +137,18 @@ class HtmlBlock extends LitElement {
 			:host([hidden]) {
 				display: none;
 			}
-			::slotted(*) {
+			:host([no-deferred-rendering="true"]) div.d2l-html-block-rendered {
+				display: none;
+			}
+			:host([no-deferred-rendering="false"])::slotted(*) {
```

Comment:
Similarly, the visual diff tests are failing because sometimes this attribute doesn't get added at all and sometimes it does, but with the explicit value `false`. I feel like I'm missing something simple here, so I'll circle back tomorrow and see if I can make heads or tails of this.